### ISSUE TITLE
Cannot resolve 'file' or 'directory' {A} in {A}

### ISSUE THREAD

**Issue by vladmiller** [webpack/webpack#2245]
Hello, I have a very weird bug with webpack

``` bash
vladmiller:resumeok vladmiller$ webpack .
Hash: 3d8a054c947d93fef16a
Version: webpack 1.12.14
Time: 4552ms
    Asset     Size  Chunks             Chunk Names
bundle.js  2.48 kB       0  [emitted]  main
   [0] multi main 40 bytes {0} [built] [1 error]
    + 1 hidden modules

ERROR in multi main
Module not found: Error: Cannot resolve 'file' or 'directory' /Users/vladmiller/Projects/alexkravchenko/resumeok in /Users/vladmiller/Projects/alexkravchenko/resumeok
 @ multi main
```

index.js is empty and my webpack.config.js is 

``` javascript
var webpack = require('webpack');
var HtmlWebpackPlugin = require('html-webpack-plugin');

var config = {
  entry: './client/index.js',

  output: {
    path: __dirname + '/dist',
    filename: 'bundle.js',
  },

  module: {
    loaders: [{
      test: /\.js$/,
      loader: 'babel',
    }, {
      test: /\.html$/,
      loader: 'html?attrs[]=img:src&attrs[]=portfolio-item:preview-url&attrs[]=portfolio-item:image-url',
    }, {
      test: /\.(scss|css)$/,
      loader: 'style!css!sass',
    }, {
      test: /\.(jpe?g|png|gif|svg)$/,
      loader: 'url?limit=25000',
    }, {
      test: /\.(svg|woff2?|ttf|eot)(\?v=\d+\.\d+\.\d+)?$/,
      loader: 'url?limit=10000',
    }]
  },

}

module.exports = config;
```

**Comment by chevex** [comment]
Same issue here. I've spent about four hours trying to debug to figure out what is going on and I'm not any closer than when I started.

**Comment by sokra** [comment]
Omit the dot after the call: `webpack .` -> `webpack`

**Comment by vladmiller** [comment]
@sokra indeed it's worked, thank you. Wondering what is the difference between two methods? Both of them seems logical.

**Comment by chevex** [comment]
Yeah I feel stupid. I spent HOURS trying to figure out why webpack suddenly stopped working when it had been working fine for months and nothing had changed. I finally found a helpful flag `webpack . --show-error-details` which showed that webpack was looking for an index file in the current directory. That's when I realized I just needed to invoke it by itself and it would pick up the webpack config. It had just been so long since I used it that I instinctively tried to invoke it on the current directory.

My suggestion would be that webpack should first look for a webpack config file in the directory if it's given a directory as the path. If I invoked it on `./index.js` then don't but if I invoke it on just `.` then I intuitively expect it to parse the config and do it's thing.

Not a huge deal but at the very least the error is really cryptic without the not-so-intuitive `--show-error-details` flag. Even then it wasn't immediately obvious why it wasn't working.
